### PR TITLE
add quicksilver testnet

### PR DESCRIPTION
**Pull request by doggystylez**

### CODE REVIEW

File: testnets/quicksilvertestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "quicksilver",
```
Comment (JeremyParish69) [inline testnets/quicksilvertestnet/assetlist.json:3]: please change to 'quicksilvertestnet'

File: testnets/quicksilvertestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "quicksilver",
```
Comment (JeremyParish69) [inline testnets/quicksilvertestnet/chain.json:3]: please change to 'quicksilvertestnet'

File: testnets/quicksilvertestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "explorers.guru",
+      "url": "https://testnet.quicksilver.explorers.guru",
+      "tx_page": "https://testnet.quicksilver.explorers.guru/transaction/${txHash}",
+      "account_page": "https://testnet.quicksilver.explorers.guru/account/${accountAddress}"
+    }
+  ]
```
Comment (JeremyParish69) [inline testnets/quicksilvertestnet/chain.json:78]: care to add the qck.svg as a chain logo? just as logo_URIs, same as for asset token logos.

**Comment by doggystylez** [comment]
changes made per request

**Pull request merged**